src/Bicep.Cli.IntegrationTests/BuildCommandTests.cs

```diff
@@ -272,6 +302,25 @@ public async Task Build_Invalid_SingleFile_ToStdOut_ShouldFail_WithExpectedError
             error.Should().ContainAll(diagnostics);
         }
 
+        // [DataTestMethod]
```

Comment:
Commenting out temporarily while I figure out how to add invalid bicepparam sample file